Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManagerInternalInterface.h

```diff
@@ -42,6 +42,11 @@ namespace AzToolsFramework
         //! @return A successful outcome object, or a string with a message detailing the error in case of failure.
         virtual MenuManagerOperationResult QueueRefreshForMenusContainingAction(const AZStd::string& actionIdentifier) = 0;
 
+        //! Queues up all menus containing the sub-menu provided for a refresh at the end of this tick.
```

Comment:
suggestion: It might be clearer to remove 'provided' here to make the sentence a little clearer